redbot/cogs/permissions/permissions.py

```diff
@@ -294,6 +294,9 @@ def __init__(self, bot: Red):
 
         `<who_or_what>` is one or more users, channels or roles the rule is for.
         """
+        if not who_or_what:
+            await ctx.send_help()
```

Comment:
This can possibly be made more clear later, but as this is more a hotfix for a last minute addition to 3.1.3, we can worry about improving the UX on this later.